policies/accepted/1.md

```diff
@@ -0,0 +1,3 @@
+# Creation and modification of proposals
+
+Any of active policies in this repo with the <ins>exception of this policy</ins> can be modified and more policies can be created at a later stage, this will require a large majority (75%) of **<ins>all active org members**</ins>' votes before it goes into effect.
```

Comment:
```suggestion
Any of the active policies in this repo with the <ins>exception of this policy</ins> can be modified and more policies can be created at a later stage, this will require a large majority (75%) of **<ins>all active org members**</ins>' votes before it goes into effect.
```